go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/gookit/color v1.5.2
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/kubeshop/testkube-operator v1.8.1
+	github.com/kubeshop/testkube-operator v1.8.0-beta001
```

Comment:
Should there be an operator release first, so we don't have beta dependencies on main?

Reply:
yes, I will update it